javascript/script.js

```diff
@@ -187,6 +187,59 @@ mainForm.addEventListener('submit', (event) => {
   event.preventDefault();
 
   if (emailValid === true) {
+    localStorage.removeItem('contactFormInput');
     mainForm.submit();
   }
 });
+
+/*
+Local Storage
+*/
+const contactFormInput = {
+  fullNameValue: '',
+  emailValue: '',
+  textareaValue: '',
+};
+
+const fullNameInput = document.getElementById('full-name');
+const emailInput = document.getElementById('email-form');
+const textareaInput = document.getElementById('contact-form-text-area');
```

Comment:
- [ ] [Optional] As the previous reviewer suggests you should add all the values of the form in a single local storage variable e.g:
```
const contactFormInput = {
  fullNameInput: document.getElementById('full-name').value,
  emailInput: document.getElementById('email-form').value,
  textareaInput = document.getElementById('contact-form-text-area').value,
};
```

Reply:
@Ismailco Hey Ismailco, thanks for reaching back, if that comment is with the [Optional] prefix, could the project be approved still? I believe is a good recommendation but, as the previous one, was not exactly described in the project requirements, therefore we believe we could implement it that way. 

Changing that only would make us change a lot of other things in the project that are could stay the way they are. 

Thanks in advance!

Reply:
@ulisesedu7, it's a requirement to use only a single object to store the data from the entire form

![Screenshot_20220513_234538](https://user-images.githubusercontent.com/25227022/168398343-8cd4ae56-2c2d-47a2-8876-c25d226253e1.png)

Reply:
@ulisesedu7, Sorry I didn't include this in the review but you should remove all the commented code if it is not been used. This will make your coded more clean and easy to read.

Reply:
Thanks @Ismailco !!